src/arch/cuda/hybridAPI/hapi_impl.cpp

```diff
@@ -887,20 +940,26 @@ static inline void profileWorkRequestEvent(hapiWorkRequest* wr,
         printf("[HAPI] invalid event during profileWorkRequestEvent\n");
     }
   }
+  else {
+    printf("[HAPI] instrumentation not initialized!\n");
```

Comment:
CmiPrintf

Reply:
Will submit a separate patch to change all printf to CmiPrintf.